DLP-6 Landing Page

KristofLevente opened the pull request.

KristofLevente committed:
DLP-5 Forgot password

KristofLevente committed:
Merge branch 'feature/DLP-5' into develop

KristofLevente committed:
DLP-6 Landing page

ptlevi commented on app/Models/Patient.php:
```diff
@@ -0,0 +1,36 @@
+//        return $this->belongsTo(User::class);
+//    }
+
+    public function user(): BelongsToMany
```
When you refer to multiple instances, you should name the function in the plural form.

ptlevi commented on app/Models/Patient.php:
```diff
@@ -0,0 +1,36 @@
+//    public function user():BelongsTo
+//    {
+//        return $this->belongsTo(User::class);
+//    }
```
You shouldn't leave commented codes.

ptlevi commented on database/migrations/2024_04_09_084812_create_patients_table.php:
```diff
@@ -0,0 +1,31 @@
+        Schema::create('patients', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained()->cascadeOnDelete();
+            $table->string('patient_name');
```
When you are in the patients table, you don't have to name a column as `patient_name`. A simple `name` is enough because you know that it relates to this table.

ptlevi commented on database/migrations/2024_04_09_115053_create_appointments_table.php:
```diff
@@ -0,0 +1,34 @@
+            $table->string('service_type');
+            $table->dateTime('appointment_date_begin');
+            $table->dateTime('appointment_date_end');
+            $table->string('appointment_type');
```
Same here. You are in the appointments table, so you don't need the appointment prefix in the columns.

ptlevi commented on database/seeders/DatabaseSeeder.php:
```diff
@@ -18,5 +21,21 @@ public function run(): void
         //     'name' => 'Test User',
         //     'email' => 'test@example.com',
         // ]);
```
Don't leave useless comments in the code

ptlevi commented on routes/web.php:
```diff
@@ -24,16 +25,23 @@
-})->middleware(['auth', 'verified'])->name('dashboard');
+//Route::get('/dashboard', function () {
+//    return view('dashboard');
+//})->middleware(['auth', 'verified'])->name('dashboard');
```
Useless comments

ptlevi commented on resources/views/layouts/navigation.blade.php:
```diff
@@ -39,15 +39,15 @@
+{{--                                                this.closest('form').submit();">--}}
+{{--                                {{ __('Log Out') }}--}}
+{{--                            </x-dropdown-link>--}}
+{{--                        </form>--}}
```
If we don't want it anymore, then delete it.

ptlevi commented on resources/views/layouts/app.blade.php:
```diff
@@ -16,7 +16,8 @@
     <body class="font-sans antialiased">
         <div class="min-h-screen bg-gray-100">
-            @include('layouts.navigation')
+{{--            @include('layouts.navigation')--}}
```
useless

KristofLevente committed:
Fixing comments

-deleted unnecessary comments from code
-made belongsToMany function plural
-changed column names in migration files and at places where the column name was used
-used translations
